stepactions/eaas-create-ephemeral-cluster-hypershift-aws/0.1/eaas-create-ephemeral-cluster-hypershift-aws.yaml

```diff
@@ -6,7 +6,8 @@ spec:
   description: >-
     This StepAction provisions an ephemeral cluster using Hypershift with 3 worker nodes in AWS.
     It does so by creating a ClusterTemplateInstance in a space on an EaaS cluster.
-  image: registry.redhat.io/openshift4/ose-cli@sha256:15da03b04318bcc842060b71e9dd6d6c2595edb4e8fdd11b0c6781eeb03ca182
+  image: public.ecr.aws/bitnami/kubectl:1.31.2@sha256:df433ab6de9adfb0223df79167719099726f946a68c1784d7739a8b6b63cd137
```

Comment:
There is a list of allowed registries for step images: https://github.com/release-engineering/rhtap-ec-policy/blob/ba7c0a249386ac15df6495e31140de7fb038187f/data/rule_data.yml#L10

Side note, we'll need to extend the CI check to take StepActions into account as well

Reply:
> Side note, we'll need to extend the CI check to take StepActions into account as well

Filed https://issues.redhat.com/browse/EC-1010

Reply:
Ah, good point. I'll have a look for an alternate image from those allowed registries/orgs.

Reply:
`quay.io/konflux-ci/appstudio-utils` might work, it contains both `yq` and `kubectl` and other tasks already use it